atuin-client/src/database.rs

```diff
@@ -458,6 +469,32 @@ impl Database for Sqlite {
             }
         };
 
+        filter_options
+            .exit
+            .map(|exit| Some(sql.and_where_eq("exit", exit)));
```

Comment:
I think you don't need the `Some` in this and the following lines

Reply:
Indeed, my mistake! It was some leftovers while trying to use and_then.
Removed on the last commit!
By the way, did you see https://github.com/ellie/atuin/commit/9597ad5faad8804720a0e23c618b537cd9942b3c#r91116514 about `interim::Dialect::Uk` ?